tests/unit/module_utils/test_files/networking-definition-valid-all-tools-dual-stack.yml

```diff
@@ -4,12 +4,6 @@ networks:
     network-v6: "fdc0:8b54:108a:c949::/64"
     gateway-v4: "192.168.122.1"
     gateway-v6: "fdc0:8b54:108a:c949::1"
-    dns-v4:
```

Comment:
Why is this file touched? Same goes for all the content in the unit tests folder. They will be moved for sure, but changed?

Reply:
I messed it up when resolving conflicts, let me put it back